extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -46,7 +46,7 @@ export class AmpIframe extends AMP.BaseElement {
         this.element);
     const containerUrl = parseUrl(containerSrc);
     assert(
-        !((' ' + sandbox + ' ').match(/\s+allow-same-origin\s+/)) ||
+        !((' ' + this.sandbox_ + ' ').match(/\s+allow-same-origin\s+/)) ||
```

Comment:
I know it is a bit hypocritical because of the usage of `this.element`, but I passed the sandbox as an argument to make this more of a pure function. No need for it to depend on some magiv instance var. Much easier to reason about then.